ligo/src/registryDAO.mligo

```diff
@@ -217,6 +217,21 @@ let registry_DAO_decision_lambda (proposal, extras : proposal * contract_extra)
         | None -> new_ce
       in (([] : operation list), new_ce)
 
+// A custom entrypoint to fetch values from Registry
+let lookup_registry (params, config_store : bytes * configured_storage) : operation list * storage =
```

Comment:
In `types.mligo`, your function corresponds to a `storable_entrypoint`, and your return type is `return`. I'm not sure if OCaml syntax allows you to use `storable_entrypoint` as a type here somehow, but I think it would be best to at least change the return type to `return`.

Reply:
>if OCaml syntax allows you to use 

Not sure if I have missed something, but it appears that it not straightforward to do so. But I have changed to use  `return` here.

Reply:
Yeah, I wasn't sure about it either. I just saw the existence of that type and assumed it could be used there.